e2e/cypress/integration/integrations/incoming_webhook/long_url_embedded_image_spec.js

```diff
@@ -0,0 +1,86 @@
+                cy.get('#showMoreButton').should('be.visible').and('have.text', 'Show more').click().then(() => {
+                    // * Assert that the Show less button is visible after Show more is clicked
+                    cy.get('#post-list');
+                    cy.get('#showMoreButton').scrollIntoView().should('have.text', 'Show less');
+                });
+            });
+        });
+    });
+});
+
+const url = 'http://localhost:8065/api/v4/image?url=http%3A%2F%2Fvia.placeholder.com%2F300.png%3Fexpires%3D213134234234234234234234%26signature%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26something%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26anything%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26a%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26b%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26c%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26d%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26f%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26g%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26h%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg%26i%3Dlksdfljkasdflkjdflknsdlknsdfljksefkjesdflknsdflknsdflknsefjkwerflkneflnsdflknsdflnksdlvnsdlvnsdvnsdvnelfkjnelfknsdvnsdflnvsdflkseflmgsdflmvsdmfnvsdfvsdfvsdfv-sdfsdflkmsdflkmsdflksdflmsdflsdlfk~msdflmsdflsdfsdffgfg';
```

Comment:
- I'd suggest to move the `url` inside the `it` block.
- Looking at the TM4J, it looks like the URL queries don't need to be an exact value of the test data. It should just be long enough for testing. So, it can be simplified to,
```javascript
const letters = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ';
const queries = letters.split('').reduce((acc, letter) => {
    const newValue = acc + `&${letter}=${letters}`;
    return newValue;
}, '');
const url = `http://via.placeholder.com/300.png?expires=213134234234234234234234${queries}`;
```